crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -333,6 +339,7 @@ impl CheckpointExecutor {
                             "Received checkpoint summary from state sync"
                         );
                         checkpoint.report_checkpoint_age(&self.metrics.checkpoint_contents_age, &self.metrics.checkpoint_contents_age_ms);
+                        self.backpressure_manager.update_highest_certified_checkpoint(*checkpoint.sequence_number());
```

Comment:
Based on the impl in `CheckpointExecutor`, is it logically possible to have `highest_certified_checkpoint` < `highest_executed_checkpoint`?

Reply:
checkpoint executor never executes uncertified checkpoints, so no